table/cs-lru.go

```diff
@@ -62,7 +62,7 @@ func (l *CsLRU) BeforeUse(index uint64, data *ndn.Data) {
 func (l *CsLRU) EvictEntries() {
 	for l.queue.Len() > csCapacity {
 		indexToErase := l.queue.Front().Value.(uint64)
-		l.cs.eraseCsDataFromReplacementStrategy(indexToErase)
+		l.cs.eraseCsDataFromReplacementStrategy(indexToErase) // TODO: fix this line with public method
```

Comment:
Not sure if this should be an exported method or not?

I also had to add this non-exported method to the `PitCsTable` interface. This is probably something that we can clean up or give a better name to.

Reply:
Agree.